components/eamxx/docs/overrides/main.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block announce %}
+EAMxx SCREAM is not supported yet. Read more about this here.
```

Comment:
I would not add "SCREAM" here. SCREAM is more of a resolution/compset combo, while the model itself is EAMxx.

Reply:
Also, what do you mean with "Read more about this here"?